src/network/core_utils.rs

```diff
@@ -1,14 +1,21 @@
 use futures::stream::TryStreamExt;
+use futures::StreamExt;
```

Comment:
are we using this import anywhere. I am not able to see in the diff.

Reply:
Yes I it need for `let mut response = match handle.clone().request(req)`, do not ask me why.
Also the linter fails with unused imports so we should be safe.

Reply:
ah i see call to request returns a future `Stream`